### PR TITLE
Ship via Fastlane

### PR DESCRIPTION
**Pull request by orta**
I think this is everything we'll need to ship a weekly build of the native side of Emission to Testflight. 

Documentation is in https://github.com/artsy/artsy.github.io/issues/345

### CODE REVIEW

**Comment by orta** [comment]
Travis or GitHub are having issues, going to finish this up tomorrow.

**Comment by orta** [comment]
This is green, and working 👍 

Slack:
screen shot 2017-05-11 at 10 29 07

Tag:
screen shot 2017-05-11 at 10 29 35

File: fastlane/Fastfile [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  unless ENV['SLACK_URL']
+    raise "You need to set SLACK_URL (#{ENV['SLACK_URL']}) in your environment."
+  end
+end
+
+# If the weekly task fails, then ship a message
+error do |_, exception|
+  # slack message: "Error Deploying Emission: #{exception}",
+  #       success: false,
+  #       payload: { Output: exception.error_info.to_s }
```
Comment (alloy) [inline fastlane/Fastfile:83]: Why not also enable this?

**Comment by alloy** [comment]
Dope 👏

File: fastlane/Fastfile (hunk 1 above)
Comment (orta) [reply fastlane/Fastfile:83]: oh yeah, I was meant to undo this (otherwise you'd be getting spammed in slack while I got it green)

Comment (alloy) [reply fastlane/Fastfile:83]: Hah, good point

**Pull request merged**